lib/boku/client.py

```diff
@@ -81,6 +83,9 @@ def api_call(self, path, params):
         )
 
         response = self._get(url)
+        if response.status_code == 204:
+            # No point in trying parse the body of 204 responses.
```

Comment:
it wouldn't be the first time if a vendor sent the wrong http code. Could you add some logging here so we know why it was ignored?